roles/jd-server/config-examples/jds-config-local-example.toml

```
@@ -1,3 +1,6 @@
+# Async Job Support
+async_mining_allowed = true
```

as suggested by @lorbax :https://github.com/stratum-mining/stratum/pull/1035#issuecomment-2252334208

> the standard behavior is to have the JDS work with that flag unset, and async_mining_allowed should offered optionally. Not having this may expose the pool to this https://github.com/stratum-mining/stratum/issues/1053 attack (pointed out in https://github.com/stratum-mining/stratum/issues/1053#issuecomment-2230607082 comment).

```suggestion
async_mining_allowed = false
```